From a524eda41dda4540e0103458a21ba2ecf8cf0196 Mon Sep 17 00:00:00 2001
From: nusu-github <29514220+nusu-github@users.noreply.github.com>
Date: Thu, 17 Mar 2022 13:44:48 +0900
Subject: [PATCH] bat file character encoding bug fixed

---
 AmatsukazeGUI/ViewModels/MakeScriptViewModel.cs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AmatsukazeGUI/ViewModels/MakeScriptViewModel.cs b/AmatsukazeGUI/ViewModels/MakeScriptViewModel.cs
index 749f163..13e0925 100644
--- a/AmatsukazeGUI/ViewModels/MakeScriptViewModel.cs
+++ b/AmatsukazeGUI/ViewModels/MakeScriptViewModel.cs
@@ -217,7 +217,7 @@ namespace Amatsukaze.ViewModels
 
             try
             {
-                File.WriteAllText(saveFileDialog.FileName, sb.ToString(), Encoding.Default);
+                File.WriteAllText(saveFileDialog.FileName, sb.ToString(), Encoding.GetEncoding(0));
             }
             catch (Exception e)
             {
-- 
2.35.1.windows.2

